tools/@aws-cdk/node-bundle/.projenrc.js

```diff
@@ -0,0 +1,35 @@
+const { typescript } = require('projen');
```

Comment:
Is projen even gonna work here?

Reply:
Yeap, I had to do some tweaks but basically yes.